modules/azure/aks/aks.tf

```diff
@@ -59,6 +59,10 @@ resource "azurerm_kubernetes_cluster" "this" {
     enable_auto_scaling          = false
     node_count                   = var.aks_config.production_grade ? 2 : 1
     only_critical_addons_enabled = true
+
+    #kubelet_config {
+    #  pod_max_pid = 0
+    #}
```

Comment:
Remove or uncomment.

Reply:
Do we think it's worth doing?
This will force a new creation of the default node pool.

I wanted to verify without having to do the migration and that is why i commented out.